Policy based traffic selectors are advertising address spaces

JackStromberg opened the pull request:
Policy based traffic selectors are advertising the VNET's address spaces, not each individual subnet.

azurecla commented:
Hi __@RobdeJong__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

JackStromberg committed:
Policy based traffic selectors are advertising address spaces

Policy based traffic selectors are advertising the VNET's address spaces, not each individual subnet.
